app/assets/stylesheets/card.scss

```diff
@@ -11,15 +12,22 @@
       color: $govuk-text-colour;
     }
 
+    .hf-card-details h2 {
+      color: $light-blue-link;
+    }
+
     .hf-card-details h3 {
       color: $light-blue-link;
     }
 
     &:hover .hf-card-container {
-      background-color: govuk-organisation-colour('department-for-education', false);
-
-      .hf-card-details p, h3 {
-          color: govuk-colour('white');
+      background-color: govuk-organisation-colour(
```

Comment:
It looks like this hover colour is not working as expected for the text and description (the text should be showing as white within the card)